2022-12-14-dnsmasq-dhcp-monitoring.md

```diff
@@ -0,0 +1,101 @@
+---
+slug: dnsmasq-dhcp-monitoring
+title: How to monitor and troubleshoot Dnsmasq for DHCP?
+description: Monitoring and troubleshooting Dnsmasq for DHCP.
+authors: satya
+tags: [how-to, monitor, troubleshoot, dnsmasq,  dns, domain-name-server, DNS-management, dhcp, network-management]
+keywords: [how-to, monitor, troubleshoot, dnsmasq,  dns, domain-name-server, DNS-management, dhcp, network-management]
+image: https://user-images.githubusercontent.com/96257330/207592157-9ec046b1-2ad0-46b8-87d1-c3e91b7dbb35.png
+
+
+---
+
+Find out how to effectively and easily monitor and troubleshoot Dnsmasq for DHCP using Netdata
+
+<!--truncate-->
+## What is Dnsmasq for DHCP?
+
+[Dnsmasq](https://thekelleys.org.uk/dnsmasq/doc.html) is an open-source, lightweight, DNS caching and forwarding server. It is designed to provide DNS resolution for small and home networks. Dnsmasq provides local DNS caching, forwarding, and recursive lookups, as well as DHCP, TFTP, and other related services. It also has support for DNS and DHCPv6, as well as various other features such as DNS-over-TLS and IPv6 privacy extensions. Dnsmasq is a versatile and highly configurable tool that is simple to use.
+DNSMasq_DHCP is a feature of DNSMasq that provides a combined server to serve both DNS (Domain Name System) and DHCP (Dynamic Host Configuration Protocol) requests. It is a fast and lightweight DHCP server with support for both IPv4 and IPv6, and can be used to serve IP addresses to hosts on a LAN. DNSMasq_DHCP also offers features such as DNS and DHCP performance tuning, DHCP address range management, and support for multiple DNS domains.
+
+This blog specifically deals with monitoring dhcp with Dnsmasq and is a followup to [DNS monitoring](https://blog.netdata.cloud/dnsmasq-monitoring/) with Dnsmasq.
```

Comment:
Small comment, but since these landing pages will soon sit somewhere else and function better AS landing pages, maybe we should remove "this blog" and say "The following sections" or "The instructions on this page deal specifically."

A few other changes made in edit and approve!

Reply:
Agree - maybe we can say something along these lines:

- For information on monitoring DHCP with DNSmasq check out <>.

Reply:
Made necessary changes.